Project.toml

```diff
@@ -12,6 +12,7 @@ NameResolution = "71a1bf82-56d0-4bbc-8a3c-48b961074391"
 
 [compat]
 julia = "1"
+JuliaVariables = "0.1.0"
```

Comment:
@ChrisRackauckas Could you change this to 0.1.3? There're several bug fixes from 0.1.0 to 0.1.3.